src/base/InstanceId.h

```diff
@@ -39,33 +39,37 @@ class InstanceId
     const char * prefix() const;
 
 public:
-    Value value = 0; ///< instance identifier
+    Value value = Value(); ///< instance identifier
 
 private:
     InstanceId(const InstanceId &); ///< not implemented; IDs are unique
     InstanceId& operator=(const InstanceId &); ///< not implemented
 };
 
-/// convenience macro to instantiate Class-specific stuff in .cc files
-#define InstanceIdDefinitions(Class, pfx) \
+/// A helper instantiation macro for (for private use only).
```

Comment:
```suggestion
/// An InstanceIdDefinitions() helper. Avoid direct use.
```

... and remove the second comment line.